docs/config.md

```diff
@@ -258,26 +264,34 @@ objectives:
 The above config will multiply obtained throughput and latency for each
 measurement by 0.5 (i.e. 0.5 * latency + 0.5 * throughput) and use this as
 the "score" to sort the measurements according to. This mode can be useful
-if you have a more advanced criteria upon which you want to sort the results.
+if a more advanced criteria needs to be used for sorting.
 
 An extension of the above `objectives` is explicitly specifying the weights.
 For example:
 ```yaml
 objectives:
-    perf_latency: 10
-    perf_throughput: 15
+    perf_latency: 2
+    perf_throughput: 3
 ```
 
 The score for each measurement will be a weighted average using the weights specified here.
 
-Similar to `<constraint>`, `<objective>` can be specified both globally and on a
-per model basis.
+`<objective>` can be specified both globally and on a per model basis.
 
-## `<parameter>`
+## Test Configuration `<parameter>`
+
+A user can specify a range of test configurations that Model Analyzer will
+profile over. The possible configuration parameters are `concurrency` and
+`batch_sizes`. One or more parameters are specified per model only. Parameters
+cannot be specified globally.
+
+Options available under this parameter are described in table below:
+
+| Option Name   | Description                                                                                                          |
+| :------------ | :------------------------------------------------------------------------------------------------------------------- |
+| `concurrency` | Request concurrency used for generating the input load. Can be a `<range>`, `<comma-delimited-list>`, or a `<list>`. |
```

Comment:
Create a third column `supporting types` to list the `` `<range>`, `<comma-delimited-list>`, or a `<list>` ``